okta/help.md

```diff
@@ -27,6 +27,49 @@ The connection configuration accepts the following parameters:
 
 ### Actions
 
+#### Push MFA Challenge
+
+This action pushes a MFA challenge to a user's device and waits for a success or rejection.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|
+|----|----|-------|--------|-----------|----|
+|factor_id|string|None|True|The factor id of the user you want to push a verify to|None|
+|user_id|string|None|True|The user id of the user you want to push a verify to|None|
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|factor_status|string|False|The factor status returned by Okta for a user|
+
+Example Output:
+
+```
```

Comment:
Add example output from a successful run from the command-line, 2 space indented.

Reply:
👍

Reply:
Bump on this one.